docs/index.rst

```diff
@@ -300,11 +300,13 @@ are accepted:
     .. versionadded:: 0.12
 
 ``FREEZER_REDIRECT_POLICY``
-    The policy for handling redirects. The default is ``'follow'`` which means
-    that when a redirect response is encountered it will follow it to get the
-    content from the redirected location. ``'ignore'`` will not stop freezing,
-    but no content will appear in the redirected location. ``'error'`` will
-    raise an exception if a redirect is encountered.
+    The policy for handling redirects. This can be:
+    * ``'follow'`` (default): when a redirect response is encountered,
+      Frozen-Flask it will follow it to get the content from the redirected
+      location. Note that redirects to external pages are not supported.
+    * ``'ignore'``: freezing will continue, but no content will appear in the
+      redirecting location.
+    * ``'error'`` : raise an exception if a redirect is encountered.
```

Comment:
Do we want to mention which exception exactly?

Reply:
It's a generic ValueError.
If we want people to catch the exception, it should have a dedicated type; if we don't, no need to document it.

Reply:
But according to the test it's a RuntimeError? 🤔 Which [isn't related to ValueError](https://docs.python.org/3/library/exceptions.html#exception-hierarchy). Did you mean to write RuntimeError in your previous comment, or is it a discrepancy?

Reply:
Oh, I thought you're commenting on `FREEZER_REDIRECT_POLICY='error'` here.

The `RuntimeError` for external redirects actually comes from deep within `werkzeug`; I don't think it was a conscious choice in Frozen-Flask. So here, I'm being vague with "not supported", rather than even mentioning you'll get an exception.

Reply:
Yup, I understood that, I just noticed that once you mentioned ValueError, but the other time RuntimeError. Since the test verifies it's RuntimeError and we're not specifying the type of the exception anyway for the reasons you mentioned, I think it was just a typo:

> It's a generic ValueError.

Thanks for the PR!